sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -20,11 +20,16 @@ public class PersonalizerClient
         private readonly HttpPipeline _pipeline;
         private readonly bool _isLocalInference;
         private string stringEndpoint;
-        private string apiKey;
+        private AzureKeyCredential _azureKeyCredential;
```

Comment:
Please try to be consistent with naming conventions. Either all private variables have an underscore or none of them do. Underscore is not a CSharp standard so I would suggest leaving them out.

Reply:
removed _ for private variables and did cleanup